pytest_console_scripts/__init__.py

```diff
@@ -124,6 +127,89 @@ def print(self) -> None:
         print('# Script stderr:', self.stderr, sep='\n')
 
 
+def _handle_command_args(
+    command: str | os.PathLike[str] | Sequence[str | os.PathLike[str]],
```

Comment:
I wonder if it makes sense to make a type alias for `str | os.PathLike[str]`, something like `PathType = ...` -- it would make things more compact and easier to understand here. We could also use it in the `isinstance`'s below.

Reply:
Yes, I think the code becomes a bit easier to read this way because we'll just be dealing with `_StrOrPath`'s and sequences of `_StrOrPath`'s. Also I think we can just give it a meaningful name like `_Path` (underscore makes sense because it's for internal use) because all of those things are basically paths. Then we'd end up with signatures like `handle_command_args(command: _Path | Sequence[_Path], ...) -> Sequence[_Path]` and it seems easier to understand.

Reply:
Those aren't just paths, those are command line arguments passed to argv.  Handling PathLike's is a convenience supported by subprocess.

Reply:
I've added the `_Command` and `_StrOrPath` type aliases, but these can not be used with `isinstance` before Python 3.10.